sympy/sets/sets.py

```diff
@@ -885,6 +886,15 @@ def right_open(self):
         """
         return self._args[3]
 
+    @property
+    def is_empty(self):
+        if self.start < self.end:
```

Comment:
This should be `if (self.start < self.end) == True:`. Otherwise you get this:
```
>>> from sympy.abc import a, b
>>> J = Interval(a, b)
>>> J.is_empty
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy/sets/sets.py", line 891, in is_empty
    if self.start < self.end:
  File "sympy/core/relational.py", line 229, in __nonzero__
    raise TypeError("cannot determine truth value of Relational")
TypeError: cannot determine truth value of Relational
```

Reply:
Thanks @jksuom .
All the changes have been implemented.